public/scripts/upload.js

```diff
@@ -34,10 +34,17 @@ async function runWorker(worker) {
       window.localforage
         .setItem("pictureArray", e.data)
         .then(function (pictureArray) {
+          if (e.data["error"] !== undefined) {
+            console.log("Get error while generating mockup", e.data["error"]);
+            window.viewModel.cancelMockup();
+            setTimeout(() => {
+              alert("Generation failed, please reload and try again later");
```

Comment:
> try again later

Seems if user try again, most likely will fail again.

Maybe we should ask them to 

1. `Try another image/device`
2. `Please file an issue at https://github.com/oursky/mockuphone.com/issues`?
3. both?

Reply:
@pkong-ds how about
```
Generation failed,
Please try another image or device, and if it still fails, please file an issue at https://github.com/oursky/mockuphone.com/issues
```
cc @hayleywhsham

Reply:
Suggestion:

```
Oops, something went wrong. Please try a different image/device. 

If it persists, report it on our GitHub: https://github.com/oursky/mockuphone.com/issues.
```

Reply:
Sound great!